apps/ex_nvr/test/ex_nvr/devices_test.exs

```diff
@@ -165,12 +199,44 @@ defmodule ExNVR.DevicesTest do
         Devices.update(device, %{
           name: @valid_camera_name,
           stream_config: %{sub_stream_uri: stream_uri},
-          settings: %{generate_bif: true}
+          settings: %{
+            generate_bif: true,
+            override_on_full_disk: true
+          }
         })
 
       assert device.name == @valid_camera_name
       assert device.stream_config.sub_stream_uri == stream_uri
       assert device.settings.generate_bif
+      assert device.settings.override_on_full_disk
+    end
+
+    test "override_on_full_disk_threshold is between 0 and 100", %{device: device} do
```

Comment:
You already tested this behaviors above, you can delete this tests.
The real tests you need to add is the `delete oldest recordings`